tests/acceptance/builds/restart-test.js

```diff
@@ -27,6 +27,6 @@ test('restarting build', function (assert) {
 
   andThen(() => {
     assert.equal(topPage.flashMessage.text, 'The build was successfully restarted.', 'restarted notification should display proper build restarted text');
-    assert.equal(buildPage.singleJobLogText, 'Hello log', 'shows log text of single build job');
+    assert.equal(buildPage.singleJobLogText, '1Hello log', 'shows log text of single build job');
```

Comment:
I'm not sure about this change. Seems like something is wrong.

Reply:
It's correct. In previous solution we had numbering in css, however it has a limitation of numbering lines when they are hidden (display:none), so numbering was moved into js. That's why log line always contains line number, even in tests.

Reply:
Ah, I see, that makes sense. Then I would suggest to make it more explicit and adapt test to actually assert that a line has both line number and text. Maybe even add an assertion to validate that numbering is correct (e.g. that second line has `2`). What do you think? Currently it just looks a bit weird and unclear.

Reply:
You are right. Now tests are extended for multiline log.